File: Java Basics - January 2020/GodzillaVSKong.java
import java.util.Scanner;
import java.util.function.DoubleFunction;

public class GodzillaVSKong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double budgetForMuvie = Double.parseDouble(scanner.nextLine());
        int numberOfStatists = Integer.parseInt(scanner.nextLine());
        double clothesFor1Statist = Double.parseDouble(scanner.nextLine());

        double dekor = budgetForMuvie*0.1;
        double clothesForAllStatists = numberOfStatists*clothesFor1Statist;

        if (numberOfStatists > 150){
            clothesForAllStatists = clothesForAllStatists*0.9;
        }
        double totalPriceFilm = dekor+clothesForAllStatists;
        if (totalPriceFilm > budgetForMuvie){
            System.out.println("Not enough money!");
            double diff = totalPriceFilm-budgetForMuvie;
            System.out.printf("Wingard needs %.2f leva more.", diff);
        }else{
            System.out.println("Action!");
            double diff2 = budgetForMuvie-totalPriceFilm;
            System.out.printf("Wingard starts filming with %.2f leva left.", diff2);

        }
    }
}

File: Python Basics - October 2020  /Conditionals /godzilla_vs_Kong.py
budget_film = float(input())
number_statists = int(input())
price_dress_1_statist = float(input())

sum_dekor = budget_film * 0.1
sum_dress = number_statists * price_dress_1_statist

if number_statists > 150:
    sum_dress = sum_dress - (sum_dress * 0.1)

total_sum_film = sum_dress + sum_dekor
money_waste = abs(total_sum_film - budget_film)

if total_sum_film > budget_film:
    print("Not enough money!")
    print(f"Wingard needs {money_waste:.2f} leva more.")

elif total_sum_film <= budget_film:
    print("Action!")
    print(f"Wingard starts filming with {money_waste:.2f} leva left.")
